src/Microsoft.AspNetCore.Mvc.Core/RedirectToActionResult.cs

```diff
@@ -23,12 +23,32 @@ public class RedirectToActionResult : ActionResult, IKeepTempDataResult
             string actionName,
             string controllerName,
             object routeValues,
+            string fragment)
+            : this(actionName, controllerName, routeValues, permanent: false, fragment: fragment)
+        {
+        }
+
+        public RedirectToActionResult(
```

Comment:
Please add doc comments for this and the other new constructor overload. No need to do the same for the existing constructor overloads -- unless you're feeling industrious! 😺